### PR TITLE
fix: use our CORS proxy as fallback

### PR DESCRIPTION
**Pull request by tjkandala**
Our CORS proxy is the default setting value, but we fall back to the public cors-anywhere demo when there's no corsAnywhereUrl in user settings.

### CODE REVIEW

File: src/snyk.ts [hunk 1]
```diff
@@ -227,11 +227,11 @@ function projectIssuesToMarkdown(project: Project, projectIssues: ProjectIssues)
...
     for (const issue of projectIssues.issues) {
         console.log(issue)
-        markdownString += `\n\n#### [${issue.issueData.title}](${issue.issueData.url}) (Priority Score: ${issue.priority.score})\n- dependency: ${
-            issue.pkgName
-        }, version${issue.pkgVersions.length > 1 ? 's' : ''} ${issue.pkgVersions.join(', ')}\n- severity: ${
-            issue.issueData.severity
-        }\n\n`
+        markdownString += `\n\n#### [${issue.issueData.title}](${issue.issueData.url}) (Priority Score: ${
+            issue.priority.score
+        })\n- dependency: ${issue.pkgName}, version${issue.pkgVersions.length > 1 ? 's' : ''} ${issue.pkgVersions.join(
```
Comment (Joelkw) [inline src/snyk.ts:232]: I assume this is unrelated to the CORS issue – just noting my review is for the CORS update.

**Review by Joelkw — APPROVED** [review]
Thanks for updating this!

File: src/snyk.ts (hunk 1 above)
Comment (tjkandala) [reply src/snyk.ts:232]: This was automatically formatted on save, so I think we're good to publish
